src/onnx/onnx.cpp

```diff
@@ -2105,6 +2106,47 @@ struct onnx_parser
         return l0;
     }
 
+    enum reduce_mode_t
+    {
+        sum,
+        mean,
+        max
```

Comment:
This should probably set the value for each of this since this converted from an integer.

Reply:
fixed